spec/javascripts/surveys-spec.js

```diff
@@ -1032,28 +1083,79 @@ describe('Surveys', function () {
   })
 
   describe('getDisplayableSurveys', function () {
-    it('does not include a survey that should not be displayed', function () {
-      spyOn(surveys, 'isSurveyToBeDisplayed').and.returnValue(false)
-      var testSurvey = {
-        startTime: new Date('July 5, 2016').getTime(),
-        endTime: new Date('July 10, 2016 23:50:00').getTime(),
-        url: 'example.com/small-survey'
-      }
+    describe('when the page is viewed on desktop', function () {
```

Comment:
I don't think we need to make changes to this test.  The implementation only relies on the result of `isSurveyToBeDisplayed` and the tests (for better or worse) mirror that via the `spyOn` calls to `isSurveyToBeDisplayed`.  It's saying, there's a bunch of logic in `isSurveyToBeDisplayed` and all this method cares about is true or false coming out of that, if you want to know more look at the tests for `isSurveyToBeDisplayed` (for which you've written a bunch more tests, yay!).  To me it feels like adding the `isBeingViewsOnMobile` stuff is redundant in this context.